### DISCUSSION TITLE
General name for resource, event, action and need.

### DISCUSSION THREAD

**Discussion by cahidenes** [bounswe/bounswe2023group2#55] [General]
This is a discussion to find a general name for resource, event, action, and need. Merve proposed `entity` in the meeting. Also ChatGPT's top 3 suggestions are `element`, `entity`, and `component`. Any further ideas?

**Comment by mehmetkuzu** [comment]
Entity is best.

**Comment by cahidenes** [comment]
Currently we use "information" and it is a good candidate as well.

**Comment by arasgungore** [comment]
While it may not be a strong contender, how about a more dynamic name like `activity`? We can name the feed which these entities appear in order as "information feed" or "activity feed" as well.

**Comment by mehmetkuzu** [comment]
I think I just got the point.
Entity is more a technical term (better in design concerns) but we need a more concrete term for the building blocks.

(Information can be confused with information coming from the field.)

**Comment by cahidenes** [comment]
I opened poll #66.
